blog/keycloak-newsletter-1.ftl

```diff
@@ -1,53 +1,53 @@
 <#--
 title = Keycloak Community Newsletter #1
-date = 2019-03-11
-publish = false
+date = 2019-03-26
+publish = true
```

Comment:
Change to publish = false and send a notification to Keycloak Team that the first newsletter is ready to be published asking if anyone has something to add. We can publish after giving folks a chance to propose additions